en/c-team-codex.md

```diff
@@ -0,0 +1,53 @@
+# C-Team Codex
+
+The C-Team in Applifting is a group of dedicated leaders who guide the company's direction and embody our core values. They ensure Applifting runs smoothly, grows sustainably, and stays true to its teal principles. By living the ideals in this codex, the C-Team drives Applifting towards its mission of creating digital products with positive impact on planetary scale.
+
+## Lead with Vision, Not Control
+
+### Trust in Teal culture
+
+Lead by setting a clear direction, not by micromanaging. Applifters should be empowered to make decisions and take responsibility.
+
+Your role is to inspire and to remove obstacles. Instead of managing people, mentor them. Help others find their path to success, be it through advice, knowledge, or resources.
+
+Value each person's contribution, regardless of title or status. Every voice in the organization should be heard.
+
+Make sure Applifting integrates all of our goals - health of the company, our wellbeing, and our vision of positive planetary impact.
+
+### Freedom with Responsibility
+
+The C-Team has full freedom to explore innovative solutions, provided they respect Applifting's values and aim to make our collective vision a reality.
+
+### Collaborate with the Council of Elders
+
+Before implementing significant changes, consult and collaborate with the Council of Elders. Discuss major decisions openly, seeking their input and guidance. Remember that the Council of Elders has veto power over C-Team's decisions - but the best veto power is the one which never has to be used.
+
+## Lead by example
+
+First and foremost, be an example for other Applifters. Live by the Applifter's codex.
+
+### Sincerity in Communication
+
+Always speak the truth and ensure honesty in every interaction, whether with team members, partners, or clients.
+
+### Own Your Failures, Share Your Successes
+
+Be accountable for both your successes and failures. Celebrate wins openly, but don’t shy away from admitting mistakes and learning from them. Vulnerability is strength. Be open to feedback from anyone in the organization, regardless of their role.
+
+### Foster Empathy and Emotional Intelligence
+
+Understanding the perspectives and emotions of others is essential. Lead with emotional intelligence, creating an environment where people feel seen and understood. Actively avoid and discourage toxic behaviors.
+
+### Preserve our Culture of Joy and Fulfillment
+
+Leadership is not only about success but also about joy. It is your role to maintain a space where people enjoy their work. Embrace a fun, informal culture where people can be themselves.
+
+### Be an Ambassador
+
+Externally, be an ambassador of Applifting's values. Communicate openly with customers and partners, ensuring your actions align with the values of transparency and authenticity.
```

Comment:
I would move this after Lead by Example, because it points to the same values but outside of the company, and Lead by Example points inside the company.